### PR TITLE
Add new parameter to enable/disable BGP route translation for NAT VirtualWan VpnGateway

### PR DESCRIPTION
**Pull request by Nilambari**
## Description
Add new parameter to enable/disable BGP route translation for NAT VirtualWan VpnGateway.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - Command let review is in progress:- https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/955
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by BethanyZhou** [comment]
Hi @Nilambari , left few comments in your cmdlet design review, please have a look.

**Comment by isra-fel** [comment]
Kindly ping @Nilambari

File: src/Network/Network/ChangeLog.md [hunk 1]
```diff
@@ -74,6 +79,9 @@
     - `Update-AzVirtualHub`
 * Updated cmdlets to expose two read-only properties of client certificate.
     - `Get-AzApplicationGatewayTrustedClientCertificate`
+* Updated below cmdlets to enable or disable BgpRouteTranslationForNat on VpnGateway.
```
Comment (wyunchi-ms) [inline src/Network/Network/ChangeLog.md:82]: Is this should move to `Upcoming Release`?

File: src/Network/Network.Test/ScenarioTests/CortexTests.ps1 [hunk 2]
```diff
@@ -748,21 +757,22 @@ function Test-CortexExpressRouteCRUD
 		$vpnClientAddressSpaces[1] = "192.168.4.0/24"
 		$updatedP2SVpnGateway = Update-AzP2sVpnGateway -ResourceGroupName $rgName -Name $P2SvpnGatewayName -VpnClientAddressPool $vpnClientAddressSpaces -CustomDnsServer 9.9.9.9 -DisableInternetSecurityFlag
 
-        	$P2SVpnGateway = Get-AzP2sVpnGateway -ResourceGroupName $rgName -Name $P2SvpnGatewayName
+        $P2SVpnGateway = Get-AzP2sVpnGateway -ResourceGroupName $rgName -Name $P2SvpnGatewayName
```
Comment (wyunchi-ms) [inline src/Network/Network.Test/ScenarioTests/CortexTests.ps1:760]: Please keep the tabs.

File: src/Network/Network/ChangeLog.md (hunk 1 above)
Comment (Nilambari) [reply src/Network/Network/ChangeLog.md:82]: Moved it under Upcoming Release section.

File: src/Network/Network.Test/ScenarioTests/CortexTests.ps1 (hunk 2 above)
Comment (Nilambari) [reply src/Network/Network.Test/ScenarioTests/CortexTests.ps1:760]: done.

**Comment by Nilambari** [comment]
@wyunchi-ms Thanks a lot for the review comments. I have incorporated them. Please take a look and help accept PR. Thanks!

**Comment by Nilambari** [comment]
@wyunchi-ms  Gentle reminder. Thanks a lot!

**Comment by Nilambari** [comment]
@wyunchi-ms hey did you get a chance to look at it? Thanks!

**Comment by Nilambari** [comment]
@wyunchi-ms Changed Base branch to main branch. FYI. Thanks!

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**